web/src/views/sleep_analysis_results/evolving_chart_scrollytelling.js

```diff
@@ -0,0 +1,316 @@
+          isInitialized={isInitialized}
+          setIsInitialized={setIsInitialized}
+          useDiv
+        />
+      </div>
+      <div style={{ marginBottom: '50%' }} />
+      <Row>
+        <Card className="shadow col-lg-6 mx-auto">
+          <CardBody>
+            <p>
+              This is your sleep time line. Each color represents one of the 5 sleep stages which are&nbsp;
+              <span className="scrollytelling_cards__w_text">Wake</span>
+              ,&nbsp;
+              <span className="scrollytelling_cards__rem_text">REM</span>
+              ,&nbsp;
+              <span className="scrollytelling_cards__n1_text">N1</span>,&nbsp;
+              <span className="scrollytelling_cards__n2_text">N2</span>
+              &nbsp;and&nbsp;
+              <span className="scrollytelling_cards__n3_text">N3</span>. Each of the colored blocks reprensent a part of
+              your night that was associated to one of these five stages. You may want to hover them as it shows more
+              details about that part of your night.
+            </p>
+          </CardBody>
+        </Card>
+      </Row>
+      <div style={{ marginBottom: '125%' }} />
+      <Row>
+        <Card className="shadow col-lg-6 mx-auto">
+          <CardBody>
+            <p>
+              The time you spent out of bed is not taken into account in this analysis. Thus, our timeline starts at the
+              bedtime, so <Metric isTime>{bedTime}</Metric> in your case, and ends at the time you got out of bed,
+              whereas&nbsp;
+              <Metric isTime>{wakeUpTime}</Metric>. Out of this <Metric isDuration>{sleepTime}</Metric>, you spent&nbsp;
+              <Metric isDuration>{efficientSleepTime}</Metric> actually sleeping. According to the American Academy of
+              Sleep Medicine, <Metric>{recommendedSleepStage}</Metric>.
+            </p>
+          </CardBody>
+        </Card>
+      </Row>
+      <div style={{ marginBottom: '125%' }} />
+      <Row>
+        <Card className="shadow col-lg-6 mx-auto">
+          <CardBody>
+            <p>
+              In total, there were <Metric>{stageShifts}</Metric> sleep stage shifts and <Metric>{awakenings}</Metric>
+              &nbsp;noctural awakenings. "High levels of sleep fragmentation, as defined by recurrent awakenings and/or
+              stage shifts may result in complaints of non-restorative sleep even when an apparently normal total sleep
+              time is present." [Shrivastava and al., 2014]
+            </p>
+          </CardBody>
+        </Card>
+      </Row>
+      <div style={{ marginBottom: '125%' }} />
+      <Row>
+        <Card className="shadow col-lg-6 mx-auto">
+          <CardBody>
+            <p>
+              You first fell asleep at <Metric isTime>{sleepOnset}</Metric>, to which we will refer to as sleep onset.
+              You woke up at <Metric isTime>{sleepOffset}</Metric>, which can also be referred to as sleep offset.
```

Comment:
```suggestion
              You woke up at <Metric isTime>{sleepOffset}</Metric>, which we will refer to as sleep offset.
```

Reply:
comme juste en haut, je propose "which will be refered to as <i>sleep offset</i>"